Statistics/Statistics.cs

```diff
@@ -1,12 +1,26 @@
-﻿using System;
+﻿  
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Statistics
 {
+    
     public class StatsComputer
     {
-        public Stats CalculateStatistics(List<float> numbers) {
-            //Implement statistics here
+        public Stats CalculateStatistics(List<double> numbers)
+        {
+            Stats obj = new Stats();
+            if (numbers.Count == 0) 
+                return obj;
+            if(numbers.Contains(double.NaN))
+            numbers.RemoveAll(double.IsNaN);
```

Comment:
This is a clean way to remove NaNs, but it will remove for the caller as well. If I were a caller of the CalculateStatistics method, I wouldn't expect my List to be modified. E.g., I may want to count the number of invalid entries after calculating statistics. Let's assume callers do not want the list to be changed.
1. Build a test to fail the current implementation. You can assert that the length of the array before and after calling are the same
2. Resolve the issue in the code.